internal/cmd/gocdk/testdata/recordreplay/Provision/out.txt

```diff
@@ -17,6 +17,9 @@ $ cd myproj
 
 $ gocdk provision add dev blob/fileblob
 gocdk: Adding "blob/fileblob" to "dev"...
+gocdk:   added a new file "fileblob.tf" to your project
+gocdk:   added a Terraform provider "local" for creating a scratch directory to "main.tf"
+gocdk:   added a BLOB_BUCKET_URL variable to "outputs.tf"
```

Comment:
Let me know what you think about these messages. I think they might be useful to users, to see what changes we are making for them, but we could also remove them and let them use `git diff` or whatever. Currently the caller of `Materialize` can decide whether to log them, by setting (or not) `MaterializeOptions.Logger` -- so, `gocdk init` does not, `gocdk provision add` does.